test/parallel/test-tick-processor.js

```diff
@@ -18,7 +19,7 @@ common.refreshTmpDir();
 process.chdir(common.tmpDir);
 // Unknown checked for to prevent flakiness, if pattern is not found,
 // then a large number of unknown ticks should be present
-runTest(/LazyCompile.*\[eval\]:1|.*%  UNKNOWN/,
+runTest(/LazyCompile.*test.js:1|.*%  UNKNOWN/,
```

Comment:
This test is designed to test the names of profiled functions appear in profiler output (as an approximation to profiler correctness). `eval` was used before since `eval` should appear on the stack every time it is observed by the profiler making it most likely to appear in the output. If the script below is now longer eval'd, we'll have to look for another symbol. `f` is too generic so maybe the function below could be renamed to something unique that the output could be checked for.